src/full/Agda/Interaction/Options.hs

```diff
@@ -109,6 +109,7 @@ data CommandLineOptions = Options
   , optGenerateVimFile  :: Bool
   , optGenerateLaTeX    :: Bool
   , optGenerateHTML     :: Bool
+  , optHTMLHighlight    :: Bool
```

Comment:
If the name is `optHTMLHighlight`, what True and False mean is ambiguous at a first glance.
```suggestion
  , optHTMLOnlyCode     :: Bool
```